src/shared/ml-thirdparty-client/models/openapi/authorizations.ts

```diff
@@ -63,3 +63,15 @@ export interface AuthorizationsPostRequest {
    */
   quote: Quote
 }
+
+export interface AuthorizationsPutIdRequest {
+  /**
+   * Authentication value provided by payer if entered.
+   */
+  authenticationInfo?: AuthenticationInfo
```

Comment:
why is authenticationInfo optional?

Reply:
yeah because the `responseType` could be equal to `REJECTED` which means the user does not provide any authentication